packages/cli-tools/bin/streamr-identity-generate.ts

```diff
@@ -1,21 +1,19 @@
 #!/usr/bin/env node
 import '../src/logLevel'
-import { KeyPairIdentity } from '@streamr/sdk/dist/types/src/identity/KeyPairIdentity'
-import { IdentityMapping, validKeyTypeValues } from '@streamr/sdk'
-import { binaryToHex } from '@streamr/utils'
+import { binaryToHex, KEY_TYPES, SigningUtil, toUserId } from '@streamr/utils'
 import { createCommand, Options } from '../src/command'
 import { formEnumArgValueDescription, createFnParseEnum } from '../src/common'
 
 createCommand()
     .description('generate a public/private key pair based on the --key-type option')
-    .requiredOption('--key-type [key-type]', `type of public/private key (${formEnumArgValueDescription(validKeyTypeValues)})`, 
-        createFnParseEnum('key-type', validKeyTypeValues))
+    .requiredOption('--key-type [key-type]', `type of public/private key (${formEnumArgValueDescription(KEY_TYPES)})`, 
+        createFnParseEnum('key-type', KEY_TYPES))
     .action(async (options: Options) => {
-        const config = IdentityMapping[options.keyType!] // required option
-        const identity = config.generate() as KeyPairIdentity
-        console.info(JSON.stringify({ 
-            publicKey: await identity.getUserId(), 
-            privateKey: binaryToHex(await identity.getPrivateKey()),
+        const signingUtil = SigningUtil.getInstance(options.keyType!) // required option
+        const keyPair = signingUtil.generateKeyPair()
+        console.info(JSON.stringify({
+            publicKey: toUserId(keyPair.publicKey),
```

Comment:
Same functionality, but more descriptive name

```suggestion
            publicKey: binaryToHex(keyPair.publicKey),
```